docs/Examples.md

```diff
@@ -285,15 +274,13 @@ splunk-operator-7c5599546c-pmbc2             1/1     Running   0          12m
 splunk-single-standalone-0                   1/1     Running   0          11m
 ```
 
-Similar to indexer clusters, you can easily scale search head clusters
-by just patching the `replicas` parameter.
+Similar to indexer clusters, you can easily scale search head clusters by just patching the `replicas` parameter.
 
 The passwords for the instance are generated automatically. To review the passwords, please refer to the [Reading global kubernetes secret object](#reading-global-kubernetes-secret-object) instructions
 
 ### Cluster Services
 
-Note that the creation of `SearchHeadCluster`, `ClusterMaster` and `IndexerCluster`
-resources also creates corresponding Kubernetes services:
+Note that the creation of `SearchHeadCluster`, `ClusterMaster` and `IndexerCluster` resources also creates corresponding Kubernetes services:
 
 ```
 $ kubectl get svc
```

Comment:
@kashok-splunk Can you speak to the port behavior for MC? Would this sample change?

Reply:
The port behavior is similar to other CR's just adding `MonitoringConsole` to above list should suffice.